LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# MIT License
```

Comment:
Not sure what dictates the license, this file or the previous one? They seem to be out of sync, too